routes/api/meetup.js

```diff
@@ -7,10 +7,10 @@ var request = require('request');
 
 exports.meetups = function (req, res) {
     'use strict';
-    var url = 'http://api.meetup.com/2/events?status=upcoming&order=time&limited_events=False&group_urlname=node-dc&desc=false&member_id=26710252&offset=0&format=json&page=20&fields=&sig_id=26710252&sig=f9a8c7e4076e5ed5825f06378a855415af3722a9';
+    var url = 'https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_urlname=node-dc&photo-host=public&page=20&fields=&order=time&desc=false&status=upcoming&sig_id=26710252&sig=dc51b92626c66cdedabf20d5c39aceb4720eb70a';
```

Comment:
@adunkman I am not really sure... might be, but that is the test URL they give me. I will do some more research and see what's actually secrete if anything.